File: code-coaches/easy/cheer-creator/cheer_creator.cpp
#include <iostream>
#include <sstream>

std::string get_cheer(int yards)
{
    if (yards > 10)
	{
		return "High Five";
	}
	else if (yards < 1)
	{
		return "shh";
	}
	else
	{
	    std::stringstream output;
		for (int i = 0; i < yards; i++)
		{
			output << "Ra!";
		}
		return output.str();
	}
}

int main()
{
	int yards;
	std::cin >> yards;

	std::cout << get_cheer(yards);

	return 0;
}

File: code-coaches/easy/cheer-creator/cheer_creator.rb
def get_cheer(yards)
  if yards > 10
    return "High Five"
  elsif yards < 1
    return "shh"
  else
    return "Ra!" * yards
  end
end

def main
  yards = gets.chomp.to_i
  puts get_cheer(yards)
end

if __FILE__ == $0
  main
end
